src/js/bellows.js

```diff
@@ -176,7 +176,7 @@
         close: function($item) {
             $item = this._item($item);
 
-            if (!$item.hasClass(cssClasses.OPENED)) {
+            if (!$item.hasClass(cssClasses.OPENED) || $item.hasClass(cssClasses.DISABLED)) {
```

Comment:
I do agree that in this case you can't really encapsulate this while incorporating the negation, but my guess is there's still a way. I think we want to go for readability whenever we can, and the point I'm trying to make here is that reducing conditional logic into a simple method with semantic meaning can go a long way.